imp_developer.py

```diff
@@ -74,11 +79,20 @@
 PR_PREPROCESSED_PREFIX   = "preprocessed."
 
 # Electric Imp settings and project properties
-EI_BUILD_API_KEY         = "build-api-key"
-EI_MODEL_ID              = "model-id"
-EI_MODEL_NAME            = "model-name"
+EI_LOGIN_KEY             = "login-key"
+EI_ACCESS_TOKEN_SET      = "access-token-set"
+EI_ACCESS_TOKEN          = "access_token"
```

Comment:
`EI_ACCESS_TOKEN          = "access_token"` -> `EI_ACCESS_TOKEN          = "token"`

Reply:
Created IMPC_ constants for mapping imp central's settings into the plugin's settings